frontend/src/context/workspaces/workspaces.tsx

```diff
@@ -108,6 +108,14 @@ export const WorkspacesProvider: FC<IWorkspacesProviderProps> = ({
       removeUserWorkspace({ workspaceId, userId })
         .then(() => {
           toast.success(`User removed successfully from workspace.`);
+          const storageWorkspace = JSON.parse(
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            localStorage.getItem("workspace")!,
+          );
```

Comment:
Can we remove the eslint disable workaround? Its preferred to not have it.
```suggestion
          const storageWorkspace = JSON.parse(
            localStorage.getItem("workspace") ?? "{}",
          );
```

Reply:
It is used in the delete operation too. I don't know if it is something necessary.